test/controllers/rental_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class RentalsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
In the main app, we strongly encourage not committing empty/generated files like this...